src/TootleLib/RayTracer/makefile

```diff
@@ -1,27 +1,27 @@
-CC 		= g++ -O3 -fpermissive -msse -D_SOFTWARE_ONLY_VERSION -D_LINUX
-#CC 		= g++ -g -Wall -fpermissive -msse -D_SOFTWARE_ONLY_VERSION -D_LINUX
+CC      = g++ -O3 -fpermissive -msse -D_SOFTWARE_ONLY_VERSION -D_LINUX
```

Comment:
Please revert all the changes in this file as makefiles do require TAB instead of spaces.